Brewfile

```diff
@@ -0,0 +1,12 @@
+brew "jemalloc"
+brew "ccache"
+brew "boost"
+brew "cmake"
+brew "thrift"
+brew "gtk-doc"
+brew "autoconf-archive" 
+brew "gobject-introspection"
+brew "git"
+brew "wget"
+brew "libtool"
+brew "lua"
```

Comment:
Is it necessary for the Brewfile to be at the top level of the repo?

Some of these dependencies are optional, and also have Category X licenses (https://www.apache.org/legal/resolved.html#category-x). We should not recommend the installation of such components if the are not needed (e.g. for C++ / Python they are not needed).

Reply:
@wesm `Brewfile` can be created at any level, just `cd` in the directory and run `brew bundle`. think that moving the specific dependencies into the specific subdirs is better?

Reply:
yes, I think that would be better. Thanks